Misc/NEWS.d/next/Library/2022-01-24-23-55-30.bpo-46491.jmIKHo.rst

```diff
@@ -0,0 +1 @@
+Allow :data:`typing.Annotated` on outside of :data:`typing.Final`/:data:`typing.ClassVar`.
```

Comment:
```suggestion
Allow :data:`typing.Annotated` to wrap :data:`typing.Final` and :data:`typing.ClassVar`.
```
Slight wording improvements. Also feel free to add "Patch by Gregory Beauregard" if you like.

Reply:
Yeah that sounds better, thanks!